modules/op-enabling-plugin.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assemblies:
+// * create/working-with-pipelines-web-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-enabling-plugin_{context}"]
+= Enabling the console
+
+To view the statistic information, you must first enable the Pipelines console plugin.
+
+.Prerequisites
+
+* You are logged on to the web console with cluster administrator permissions.
```

Comment:
Again, maybe say that it's the ocp web console (since we're talking about two different consoles in this procedure)

Reply:
Not really, it's one console, the other one is a "console plugin" and I forgot the word plugin :)